netutils/interface.py

```diff
@@ -280,8 +294,8 @@ def __hash__(self) -> int:  # noqa: D105
 class CCString(CharacterClass):
     """Strings are sorted lexicographically."""
 
-    def __lt__(self, other) -> bool:  # noqa: D105
-        return self.weight < other.weight or self.val < other.val
+    def __lt__(self, other) -> bool:  # type: ignore  # noqa: D105
```

Comment:
Can be fixed with `def __lt__(self, other: "CharacterClass") -> bool:  # noqa: D105`